### PR TITLE
refactor(Views): move views to @cerebral

### PR DESCRIPTION
**Pull request by henri-hulski**

### CODE REVIEW

File: packages/node_modules/cerebral/package.json [hunk 1]
```diff
@@ -19,7 +19,12 @@
   "homepage": "http://cerebral.github.io/cerebral-website",
   "dependencies": {
     "eventemitter3": "^2.0.2",
-    "function-tree": "next"
+    "function-tree": "next",
+    "@cerebral/react": "next",
+    "@cerebral/inferno": "next",
+    "@cerebral/preact": "next",
+    "@cerebral/vue": "next",
+    "@cerebral/angularjs": "next"
```
Comment (christianalfoni) [inline packages/node_modules/cerebral/package.json]: Should not be dependencies? The views are optional... might not even add a view :-)

Comment (henri-hulski) [reply packages/node_modules/cerebral/package.json]: I thing we need these imports so we can reexport them while the old imports are deprecated.
They will be removed when deprecation will be removed.

File: packages/node_modules/@cerebral/inferno/README.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# @cerebral/inferno
+
+[Inferno](http://infernojs.org) view for cerebral.
+
+**NPM**
+
+`npm install @cerebral/inferno inferno inferno-component inferno-create-element babel-plugin-inferno`
+
+```js
+import Inferno from 'react'
```
Comment (edgesoft) [inline packages/node_modules/@cerebral/inferno/README.md]: Is this correct?

Comment (henri-hulski) [reply packages/node_modules/@cerebral/inferno/README.md]: I just copied it from docs/api/components.md, but you're right it doesn't look correct.

**Pull request merged**